notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -128,181 +133,124 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "install_aip"
+        "id": "3b1ffd5ab768"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the latest version of Cloud Storage, Bigquery and Vertex AI SDKs for Python."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "1fd00fa70a2a"
-      },
-      "outputs": [],
-      "source": [
-        "# Install the packages\n",
-        "! pip3 install --upgrade google-cloud-aiplatform \\\n",
-        "                        google-cloud-storage \\\n",
-        "                        'google-cloud-bigquery[pandas]'"
+        "## Getting Started"
```

Comment:
change "Getting started" to "Get started" to be in compliance with style guide. I've updated the template to reflect this change